element-connector/src/main/java/org/eclipse/californium/elements/tcp/CloseOnIdleHandler.java

```diff
@@ -0,0 +1,25 @@
+package org.eclipse.californium.elements.tcp;
+
+import io.netty.channel.ChannelDuplexHandler;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.timeout.IdleStateEvent;
+import org.eclipse.californium.elements.UDPConnector;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Channel handler that closes connection if an idle event was raised.
+ */
+class CloseOnIdleHandler extends ChannelDuplexHandler {
+
+    private final static Logger LOGGER = Logger.getLogger(UDPConnector.class.getName());
```

Comment:
Fixed.

On Sun, Jun 26, 2016 at 11:42 PM Kai Hudalla <notifications@github.com>
wrote:

> In
> element-connector/src/main/java/org/eclipse/californium/elements/tcp/CloseOnIdleHandler.java
> <https://github.com/eclipse/californium/pull/61#discussion_r68528437>:
>
> > +package org.eclipse.californium.elements.tcp;
> > +
> > +import io.netty.channel.ChannelDuplexHandler;
> > +import io.netty.channel.ChannelHandlerContext;
> > +import io.netty.handler.timeout.IdleStateEvent;
> > +import org.eclipse.californium.elements.UDPConnector;
> > +
> > +import java.util.logging.Level;
> > +import java.util.logging.Logger;
> > +
> > +/**
> > + * Channel handler that closes connection if an idle event was raised.
> > + */
> > +class CloseOnIdleHandler extends ChannelDuplexHandler {
> > +
> > +    private final static Logger LOGGER = Logger.getLogger(UDPConnector.class.getName());
>
> correct logger?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eclipse/californium/pull/61/files/7df49abe1d1386555b71cd0e43a502a5dda47bb2..fe1a65e32d8faf3158e3b88505ca42a96badb39b#r68528437>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APu4MloPsp-3v8Xrmuc9fc_wCdu6z7mlks5qP3DpgaJpZM4I9Q-g>
> .
>